modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/components/export-form/ExportFormModalBody.es.js

```diff
@@ -0,0 +1,96 @@
+/**
```

Comment:
Knowing Milton, I think he would suggest to use typescript. 😛

Reply:
👀